src/Negotiation/Negotiator.php

```diff
@@ -21,23 +21,62 @@ protected function match(AcceptHeader $accept, AcceptHeader $priority, $index)
             return null;
         }
 
-        $ab = $accept->getBasePart();
-        $pb = $priority->getBasePart();
+        $acceptBase = $accept->getBasePart();
+        $priorityBase = $priority->getBasePart();
 
-        $as = $accept->getSubPart();
-        $ps = $priority->getSubPart();
+        $acceptSub = $accept->getSubPart();
+        $prioritySub = $priority->getSubPart();
 
         $intersection = array_intersect_assoc($accept->getParameters(), $priority->getParameters());
 
-        $baseEqual = !strcasecmp($ab, $pb);
-        $subEqual  = !strcasecmp($as, $ps);
+        $baseEqual = !strcasecmp($acceptBase, $priorityBase);
+        $subEqual  = !strcasecmp($acceptSub, $prioritySub);
 
-        if (($ab === '*' || $baseEqual) && ($as === '*' || $subEqual) && count($intersection) === count($accept->getParameters())) {
+        if (($acceptBase === '*' || $baseEqual)
+            && ($acceptSub === '*' || $subEqual)
+            && count($intersection) === count($accept->getParameters())
+        ) {
```

Comment:
I split this condition across multiple lines to better understand the precedence of operations.